### PR TITLE
Export map with labels

### PR DESCRIPTION
**Pull request by MikeTschudi**

### CODE REVIEW

File: src/assets/t9n/public-notification/resources.json [hunk 1]
```diff
@@ -1,38 +1,27 @@
 {
```
Comment (jmhauck) [inline src/assets/t9n/public-notification/resources.json:1]: @MikeTschudi This file is off...as far as I know[ this](https://github.com/Esri/solutions-components/blob/new-draw-tools/src/assets/t9n/public-notification/resources.json) has all the strings we need for the main PN component. If you needed to add something else could you compare with the above and see if its missing?

File: src/assets/t9n/public-notification/resources_en.json [hunk 2]
```diff
@@ -1,37 +1,27 @@
 {
```
Comment (jmhauck) [inline src/assets/t9n/public-notification/resources_en.json:1]: same as above here

**Comment by jmhauck** [comment]
@MikeTschudi the only thing I see is the mismatch with the t9n files

**Comment by MikeTschudi** [comment]
Thanks for checking. Can you review this PR, and then stomp on the strings via #210?

**Review by jmhauck — APPROVED** [review]

**Comment by MikeTschudi** [comment]
Thank you, @jmhauck

**Pull request merged**